bias in delays session: minor edits (#450)
nfidd/nfidd: sessions/biases-in-delay-distributions.qmd

# Dates, not days: censoring

So far, the data we've been using has represented time as a *continuous* variable: a numeric time since a given starting point.
Data on health outcomes are usually not recorded in this way.
Instead, we usually deal with *dates*: a discrete interval.

> We refer to this restriction of the data, where we only know that an event has occurred within a given time period (e.g. a day) but not when exactly within that period, as *censoring*.

Reviewer: ```suggestion
We refer to this restriction of the data, where we only know that an event has occurred within a given time period (e.g. a day) but not when exactly within that period, as *interval censoring*.
```

# Dates, not days: censoring

> So far, the data we've been using has represented time as a *continuous* variable: a numeric time since a given starting point. Data on health outcomes are usually not recorded in this way. Instead, we usually deal with *dates*: a discrete interval. We refer to this restriction of the data as *censoring*.

Reviewer 2: ```suggestion
So far, the data we've been using has represented time as a *continuous* variable: a numeric time since a given starting point.
Data on health outcomes are usually not recorded in this way.
Instead, we usually deal with *dates*: a discrete interval.
We refer to this restriction of the data, where we only know that an event has occurred within a given time period (e.g. a day) but not when exactly within that period, as *censoring*.
```

Each of the numbers now represent the number of days that have passed since the start of the outbreak.
That is, each of the numbers correspond to a day.
In that sense, the data is more like typical data we get from infectious disease outbreaks, where we would usually have a line list with key events such as symptom onset or death reported by a *date*.

> In statistical terms, we call the data *double interval censored*:

Reviewer 2: (if you haven't yet) double check this doesn't need a new line for it to be recognised as bulleted list (I thought it did but was wrong on the updated quarto slides)

Each of the numbers now represent the number of days that have passed since the start of the outbreak.
That is, each of the numbers correspond to a day.
In that sense, the data is more like typical data we get from infectious disease outbreaks, where we would usually have a line list with key events such as symptom onset or death reported by a *date*.
In statistical terms, we call the data *double interval censored*:
- "double" because the delays represent the time between two events that are both censored
- "interval" because all we know about the timings of the events is that they happened in a certain time interval (between 0:00 and 23:59 on the recorded day).

::: {.callout-note collapse="true"}
# Why are we so worried?

> We'll be spending a lot of time thinking about a potentially "short" 24 hour time interval. In this session, we'll start to get a sense for how variation, uncertainty, and bias within this interval might combine to impact daily outbreak estimates. The concepts and techniques that we use here also apply to longer time intervals that we might find in surveillance data, for example, reported case counts aggregated by week.

Reviewer: ```suggestion
We'll be spending a lot of time thinking about a potentially "short" 24 hour time interval. In this session, we'll start to get a sense for how variation, uncertainty, and bias within this interval might combine to impact daily outbreak estimates.
The concepts and techniques that we use here also apply to longer time intervals that we might find in surveillance data, for example, reported case counts aggregated by week.
```

# Real-time estimation: truncation

The data set we have looked at so far in this session is a "final" data set representing an outbreak that has come and gone.
However, information on delay distributions is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.

> Estimating delays in real time comes with particular challenges, as we know that our data are always incomplete: not enough time has passed to observe all of the relevant events. We call this incompleteness *truncation*. This is a problem if we try to use incomplete secondary event data (such as recent symptom onsets) to infer the pattern of primary events (such as infections) - our estimates will be biased because we are missing some of the data.

Reviewer 2: just because the data _might_ be complete (if no more events happened).
```suggestion
Estimating delays in real time comes with particular challenges, as we know that our data are usually incomplete: not enough time has passed to observe all of the relevant events that could have heppened.
We call this incompleteness *truncation*.
This is a problem if we try to use incomplete secondary event data (such as recent symptom onsets) to infer the pattern of primary events (such as infections) - our estimates might be biased because we are missing some of the data to get the full picture.
```

This bias also changes over the course of an outbreak. If, for example, infections are exponentially increasing, then there will be disproportionately more people with recent symptom onset.
Without adjustment, this would artificially decrease the estimate of the mean delay compared to its true value for all infections that have in fact already occurred.

> This happens because most infections are recent (due to the exponential increase), but later symptom onsets amongst these have not had a chance to happen yet. So among a cohort of people infected at the same time, those with shorter delays to onset would be over-represented.

Reviewer 2: ```suggestion
This happens because most infections are recent (due to the exponential increase), but later symptom onsets amongst these have not had a chance to happen yet.
So among a cohort of people with symptom onset at the same time, those with shorter delays to onset would be over-represented.
```
(the bias occurs when looking backwards)

Author: oops, thanks!

# Real-time estimation: truncation

The data set we have looked at so far in this session is a "final" data set representing an outbreak that has come and gone.
However, information on delay distributions is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.

Estimating delays in real time comes with particular challenges, as we know that our data are usually incomplete: not enough time has passed to observe all of the relevant events that could have happened.
We call this incompleteness *truncation*.

> This is a problem if we try to use incomplete secondary event data (such as recent symptom onsets) to infer the pattern of primary events (such as infections) - our estimates might be biased because we are missing some of the data to get the full picture.

Reviewer: I think this truncation def could do with a workshop. The key is needing to mention both events I.e the event we are using and the observed event. It's when we condition on a delay observed event we get right truncation of the target event.

Author: see what you think in [commit](https://github.com/nfidd/nfidd/pull/450/commits/637f81898ed59c9b4065c320ef02ac451389a0d1), or please do edit away!

# Real-time estimation: truncation

The data set we have looked at so far in this session is a "final" data set representing an outbreak that has come and gone.
However, information on delay distributions is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.

Estimating delays in real time comes with particular challenges, as we know that our data are usually incomplete: not enough time has passed to observe all of the relevant events that could have happened.

> We call this incompleteness *truncation*.

Reviewer: ```suggestion
We call this incompleteness *right truncation*.
```

Reviewer: I think we probably want to be specific here as with censoring so people aren't confused we could also give the general definition but I don't think we need to.

This is a problem if we try to use an incomplete set of data about one type of event to infer the pattern of a previous epidemiological event. 

> For example, our target might be to infer the "primary" event of infections. We want to do this using some observed data about a "secondary" event, such as recent symptom onsets. 

Reviewer: ```suggestion
For example, our target might be to infer a "primary" event, i.e. symptom onsets. We want to do this using some observed data about a "secondary" event, such as the date of report for symptom onsets. 
```
